src/test/java/task10/TestHelper.java

```diff
@@ -0,0 +1,37 @@
+package task10;
+
+import com.google.gson.Gson;
+import parser.NoSuchFileException;
+import shop.Cart;
+
+import java.io.*;
+
+public class TestHelper{
```

Comment:
Good option to have your own class with reader and writer :) 
I was expecting that it will be a bit another methods that you have in parser classes, but this option also works